### ISSUE TITLE
Adding better physics to the default parameters.

### ISSUE THREAD

**Issue by ondrejkrejci** [Probe-Particle/ppafm#233]
The default tip parameters in the [wiki](https://github.com/Probe-Particle/ppafm/wiki/Params), in the [common.py](https://github.com/Probe-Particle/ppafm/blob/main/ppafm/common.py) and in the [default params.ini](https://github.com/Probe-Particle/ppafm/blob/main/ppafm/defaults/params.ini) are basically old and they do not capture the right physics of the most common (Cu-)CO tip.

I would suggest to changing it to:
```
rProbe [0.,0. 3.0] # (Cu-O distance is really only 3 Ang)
tip 'dz2'
Q -0.1
klat 0.24
```
What do you think?

**Comment by NikoOinonen** [comment]
Makes sense. But is there any chance that it would break some existing scripts (if it even matters at this point)?

**Comment by ondrejkrejci** [comment]
> But is there any chance that it would break some existing scripts (if it even matters at this point)? 
Quick check from my side:


- charge should be maybe dicscussed: since any non-params.ini calculations will automatically want FF_el, also it will affect these tests: [_test_vdw.py](https://github.com/Probe-Particle/ppafm/blob/main/tests/_test_vdw.py); I am not sure about these ones: [human_eye/test_TipForce.py](https://github.com/Probe-Particle/ppafm/blob/main/tests/human_eye/test_TipForce.py) and [human_eye/test_splines.py](https://github.com/Probe-Particle/ppafm/blob/main/tests/human_eye/test_splines.py) and [human_eye/test_vdwDamping.py](https://github.com/Probe-Particle/ppafm/blob/main/tests/human_eye/test_vdwDamping.py)
- The pyridineDensOverlap example do not have the klat in [params.ini](https://github.com/Probe-Particle/ppafm/blob/main/examples/pyridineDensOverlap/params.ini)
- `tip` is not defined in [benzeneBrCl2/params.ini](https://github.com/Probe-Particle/ppafm/tree/main/examples/benzeneBrCl2/params.ini) and [pyridineDensOverlap/params.ini](https://github.com/Probe-Particle/ppafm/tree/main/examples/pyridineDensOverlap/params.ini) and [PTCDA_single/params.ini](https://github.com/Probe-Particle/ppafm/tree/main/examples/PTCDA_single/params.ini) and [PTCDA_Hartree/params.ini](https://github.com/Probe-Particle/ppafm/tree/main/examples/PTCDA_Hartree/params.ini) and [Graphene_mod/params.ini](https://github.com/Probe-Particle/ppafm/tree/main/examples/Graphene_mod/params.ini) and [Graphene/params.ini](https://github.com/Probe-Particle/ppafm/tree/main/examples/Graphene/params.ini) and [Graphene-spline/params.ini](https://github.com/Probe-Particle/ppafm/tree/main/examples/Graphene-spline/params.ini); CoPc-IETS has charge set to 0, so it does not matter.
- [CorrectionLoopGraphene](https://github.com/Probe-Particle/ppafm/tree/main/examples/CorrectionLoopGraphene) and [Generator](https://github.com/Probe-Particle/ppafm/tree/main/examples/Generator) do not have params.ini at all, but I do not know, if it takes those parameters into account at all.

Anything else should be OK.
